### PR TITLE
issue 12 solved

### PR DESCRIPTION
**Pull request by jHetvi**

### CODE REVIEW

**Comment by jHetvi** [comment]
In pubspec.lock
<https://github.com/pariyatti/patta/pull/17#discussion_r470139243>:

> @@ -218,6 +225,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.1.1"
+  fake_async:

I'm not sure what this new dependency (fake_async) is. Is it necessary?
------------------------------

In pubspec.lock
<https://github.com/pariyatti/patta/pull/17#discussion_r470139880>:

> @@ -113,13 +113,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0+1"
+  characters:

I'm not sure what this new dependency (characters) is. Is it necessary?


Even I don't know about this new changes as it's coming after I updated my
Flutter to the latest version. But I'll try an research and let you know
about these changes as soon as I learn about it.

And for the rest yes I will make the required changes and commit again.

File: pubspec.lock [hunk 1]
```diff
@@ -113,13 +113,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0+1"
+  characters:
```
[the remark this answers is not in the document]
Comment (jHetvi) [reply pubspec.lock:116]: This change is due to difference in Flutter version.

File: pubspec.lock [hunk 2]
```diff
@@ -218,6 +225,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.1.1"
+  fake_async:
```
[the remark this answers is not in the document]
Comment (jHetvi) [reply pubspec.lock:228]: This change is due to difference in Flutter version.

**Comment by VarunBarad** [comment]
@jHetvi @deobald don't bother with changes in `pubspec.lock` as long as no changes have been made in `pubspec.yaml`

@deobald ideally `pubspec.lock` should be ignored from VCS, I did not know about it when I was working on that project. Once this PR is merged in, I will add `pubspec.lock` to `.gitignore` and remove it from tracking.

File: config/app_config.sample.json [hunk 3]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (VarunBarad) [inline config/app_config.sample.json:1]: @jHetvi we have to copy this file to `config/app_config.json` and not move it there. We will need to keep the `config/app_config.sample.json` file available for people who work on this in the future.

Can you please add this file back?

**Comment by deobald** [comment]
Closing this PR because it's accidentally against the `master` branch in `pariyatt/patta`.

**Pull request closed without merging**